spec/javascripts/components/cookie-banner-spec.js

```diff
@@ -113,20 +113,4 @@ describe('Cookie banner', function () {
 
     expect(newCookieBanner).not.toBeVisible()
   })
-
```

Comment:
Sadness. Might we have a test? ;-)

Reply:
Sure, will think of what's a good way of testing it - raised it quickly to get feedback on.

Reply:
Updated with a test to check the cookie banner is hidden when used in an iframe (`window.parent` is different than `window`)